### PR TITLE
Identical YML configs update

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CRTX-133868)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/BluecatAddressManager/ModelingRules/BluecatAddressManager_1_3/BluecatAddressManager_1_3.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-fromversion: 8.2.0
-id: Bluecat_AddressManager
-name: Bluecat Address Manager
+fromversion: 8.4.0
+id: Bluecat_AddressManager_ModelingRule
+name: Bluecat Address Managern Modeling Rule
```
Comment (ShirleyDenkberg) [inline Packs/BluecatAddressManager/ModelingRules/BluecatAddressManager_1_3/BluecatAddressManager_1_3.yml:3]:
```suggestion
name: Bluecat Address Manager Modeling Rule
```

File: Packs/CitrixADC/ModelingRules/CitrixADC_1_3/CitrixADC_1_3.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-fromversion: 6.10.0
-id: Citrix ADC
-name: Citrix ADC
+fromversion: 8.4.0
+id: Citrix_ADC_ModelingRule
+name: Citrix ADCn Modeling Rule
```
Comment (ShirleyDenkberg) [inline Packs/CitrixADC/ModelingRules/CitrixADC_1_3/CitrixADC_1_3.yml:3]:
```suggestion
name: Citrix ADC Modeling Rule
```

File: Packs/BluecatAddressManager/ReleaseNotes/1_1_14.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### Bluecat Address Managern Modeling Rule
```
Comment (ShirleyDenkberg) [inline Packs/BluecatAddressManager/ReleaseNotes/1_1_14.md:4]:
```suggestion
##### Bluecat Address Manager Modeling Rule
```

File: Packs/CitrixADC/ReleaseNotes/1_0_4.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### Citrix ADCn Modeling Rule
```
Comment (ShirleyDenkberg) [inline Packs/CitrixADC/ReleaseNotes/1_0_4.md:4]:
```suggestion
##### Citrix ADC Modeling Rule
```

**Comment by ShirleyDenkberg** [comment]
@OBavly Doc review completed.

**Comment by eepstain** [comment]
Following to XSUP-42012, it was found that similar ID and Name objects in Modeling/Parsing Rule's YML configs caused some issues on customers dev/prod tenant. A change was needed in order to avoid such issues.

@ShahafBenYakir

**Review by OBavly — APPROVED** [review]

**Pull request merged**